composer.json

```diff
@@ -19,7 +19,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^5.3.9|~7.0",
-        "symfony/framework-bundle": "^2.3.27|^2.6.6|~3.0",
```

Comment:
I didn't expect this to lead to other results. But anyway, that would some the same issue @Tobion pointed to for the old way.